portfolio/src/main/webapp/script.js

```diff
@@ -22,6 +21,47 @@ var counter = 1;
 var intervalLength = setInterval(change, 2000);
```

Comment:
I didn't see this earlier but for all of your `var` usages, could you switch to `const` or `let` instead?